draft-ietf-quic-recovery.md

```diff
@@ -348,6 +348,11 @@ acknowledgement delays are likely to be non-repeating and limited to the
 handshake. The endpoint can therefore use them without limiting them to the
 max_ack_delay, avoiding unnecessary inflation of the RTT estimate.
 
+Ignoring max_ack_delay at the beginning of the connection can lead to a
+substantially inflated smoothed_rtt. Therefore, prior to handshake confirmation,
```

Comment:
I do not think that this sentence is correct. Ignoring `max_ack_delay` is a way to prevent inflation of SRTT. The problem here is that ignoring `max_ack_delay` is not enough, if `min_rtt` is not established.